### PR TITLE
fix(2302): Show warning when build parameters are different from default values

### PR DESCRIPTION
**Pull request by wahapo**
## Context
Now we just need to test it.

When using parameter builds, it will show a warning if users fill values that differ from the default values defined in `screwdriver.yaml`.

## Objective
When you change the value of the form from the default one defined in `screwdriver.yaml`, an icon will be displayed and the background color will change.
image

The `screwdriver.yaml` used in the image above is the following.
```
parameters:
  nameA: "value1"
  nameB:
    value: "value2"
    description: "description of nameB"
  nameC: ["value3", "value4"]

jobs:
  main:
    image: node:12
    steps:
      - ls
```

The list view and parameter list display screens have been modified as well.

## References
https://github.com/screwdriver-cd/screwdriver/issues/2302

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by yoshwata — APPROVED** [review]

File: app/components/pipeline-parameterized-build/component.js [hunk 1]
```diff
@@ -62,20 +69,27 @@ export default Component.extend({
         description: "User running build"
       }
    * ]
-   * @param  {Object} parameters [description]
-   * @return {[type]}            [description]
+   * @param  {Object} parameters        [description]
+   * @param  {Object} defaultParameters [description]
+   * @return {[type]}                   [description]
```
Comment (tkyi) [inline app/components/pipeline-parameterized-build/component.js:74]: can you update the jsdocs

File: app/helpers/array-includes.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { helper } from '@ember/component/helper';
+
+/**
+ * return whether toFind exists in array or not
+ * @method ansiColorize
```
Comment (tkyi) [inline app/helpers/array-includes.js:5]:
```suggestion
 * @method arrayIncludes
```

File: tests/integration/components/pipeline-parameterized-build/component-test.js [hunk 3]
```diff
@@ -86,10 +86,44 @@ module('Integration | Component | pipeline-parameterized-build', function(hooks)
...
+    });
+
+    await render(hbs`{{pipeline-parameterized-build
+      pipeline=pipeline
+      buildParameters=buildParameters
+      showSubmitButton=showSubmitButton}}`);
+    assert.dom('.form-group').exists({ count: 3 }, 'There are 3 parameters');
+    assert
+      .dom('.fa-exclamation-triangle')
+      .exists({ count: 2 }, 'Theare are 2 default value warnings');
```
Comment (tkyi) [inline tests/integration/components/pipeline-parameterized-build/component-test.js:124]:
```suggestion
      .exists({ count: 2 }, 'There are are 2 default value warnings');
```

**Review by tkyi — APPROVED** [review]
Small comments

**Comment by wahapo** [comment]
Thanks! I corrected the points that were pointed out.

**Review by ibu1224 — APPROVED** [review]

**Pull request merged**